default/python/AI/turn_state/__init__.py

```diff
@@ -0,0 +1,39 @@
+"""
+This module contains tools to work with pre-aggregated information about AI state.
+This information is not stored in the save game and recalculated each turn.
+
+The main purpose of this module is to separate code that aggregates information and code that makes a decision.
+Decision-making code is smart and this is the dumb module,
+that means that we try to make this module API as comfortable as possible.
```

Comment:
that means that we try to -> which means that we will try to   (or)  which means that we have tried to